quickstarts/apache/config.yml

```diff
@@ -4,14 +4,14 @@ id: ad5affab-545a-4355-ad48-cfd66e2fbf00
 name: apache
 
 # Title of the quickstart
-title: Apache
+title: Apache httpd
 
 # Description of the quickstart
 
 description: |+
-  ## Understanding Apache
+  ## Understanding Apache httpd
 
-  The Apache HTTP Server is a free and open-source, secure, efficient, and extensible HTTP web server for the Windows and UNIX operating systems.
+The Apache HTTP Server is a free and open-source, secure, efficient, and extensible HTTP web server for the Windows and UNIX operating systems.
```

Comment:
The issue may be with the spacing on this line, try tabbing the lines to be indented after the description block.

![Screen Shot 2021-11-06 at 10 20 08 AM](https://user-images.githubusercontent.com/1395158/140616534-e9d98130-747e-4b1f-9818-12cd3a170728.png)

Reply:
Fixed 🚀